### ISSUE TITLE
Labels feature with QTransmission server

### ISSUE THREAD

**Issue by LETRA** [tympanix/Electorrent#156]
**Client:**
- [ ] µTorrent
- [ ] qBittorrent
- [X] Transmission
- [ ] rTorrent
- [ ] Deluge

**Client Version:**

· QTransmission 2.94

**Operating System:**

· Qnap QTS 4.3.6 / mac OS Mojave 10.14.5

**Application Version:**

· Electorrent 2.6

**Description:**

The app is wonderful and the best we have for the Mac but the labels feature does not work. My QTransmission server in Qnap does not support that feature either, but I wanted to suggest that it be supported independently in Electorrent for a better organization of the downloads.

Thank you.

**Comment by tympanix** [comment]
You are right, Transmission does not support labels (at least not through it's API). Supporting it independently would actually be possible, but would only be usable for organisational purposes. It's not a bad idea. I will keep this issue so that we might get around to it.

Sorry for the late reply

**Comment by LETRA** [comment]
Thank you very much :)

**Comment by Buntelrus** [comment]
> You are right, Transmission does not support labels (at least not through it's API). Supporting it independently would actually be possible, but would only be usable for organisational purposes. It's not a bad idea. Actually it could apply to other clients as well, not just Transmission. I will keep this issue so that we might get around to it.
> 
> Sorry for the late reply

It seems that transmission support labels regarding to this [document](https://github.com/transmission/transmission/blob/master/extras/rpc-spec.txt). Through a quick test I can verify that I was able to add and retrieve a label.

**Comment by LETRA** [comment]
I am currently using qbittorrent to maintain compatibility and functionality with your app. But it's good news for Transmissions users and others if we want to use that client again at some point.

**Comment by tympanix** [comment]
Transmission now supports labels - changed the scope for the issue support that

**Comment by tympanix** [comment]
Fixed in #537
